neo4j/tests/common.py

```diff
@@ -1,9 +1,7 @@
-import os
-
 from datadog_checks.dev import get_docker_hostname, get_here
 
 HERE = get_here()
 HOST = get_docker_hostname()
 PORT = '2004'
 METRICS_URL = f'http://{HOST}:{PORT}/metrics'
-NEO4J_VERSION = os.getenv('NEO4J_VERSION')
+NEO4J_VERSION = '5.5'
```

Comment:
Why hardcode this? You would only be testing Neo4J 5.5.

Reply:
Can you give me an idea on how to pass that in as a parameter or someway to avoid the hard coding?
Thanks.

Reply:
@yzhan289 Made some changes to the test structure in the last update.